src/common/hooks/use-selected-asset.ts

```diff
@@ -0,0 +1,47 @@
+import { useSetRecoilState } from 'recoil';
+import { useLoadable } from '@common/hooks/use-loadable';
+import { selectedAssetIdState, selectedAssetStore } from '@store/asset-search';
+import { AssetWithMeta } from '@store/tokens';
+import { getTicker } from '@common/utils';
+import { useMemo } from 'react';
+import { ftDecimals, stacksValue } from '@common/stacks-utils';
+import BigNumber from 'bignumber.js';
+
+export function useSelectedAsset() {
+  const { value: selectedAsset } = useLoadable(selectedAssetStore);
+  const setSelectedAsset = useSetRecoilState(selectedAssetIdState);
+  const handleUpdateSelectedAsset = (asset: AssetWithMeta | undefined) => {
+    setSelectedAsset(asset?.name || undefined);
+  };
+  const name = selectedAsset?.meta?.name || selectedAsset?.name;
+  const isStx = selectedAsset?.name === 'Stacks Token';
```

Comment:
Checking for is stx by a human-readable name isn't ideal. Can we not check by symbol?